sdcm/utils/nemesis_utils/indexes.py

```diff
@@ -41,7 +42,8 @@ def is_cf_a_view(node: BaseNode, ks, cf) -> bool:
             return False
 
 
-def get_column_names(session, ks, cf, is_primary_key: bool = False, filter_out_collections: bool = False, filter_out_static_columns: bool = False) -> list:
+def get_column_names(session, ks, cf, is_primary_key: bool = False, filter_out_collections: bool = False,
```

Comment:
as this one gets more logic into it,

maybe it's worth having a unit/integration test for it

see `test_get_any_ks_cf_list` for a quick why to write such test